server/src/main/scala/net/wiringbits/repositories/daos/TablesDAO.scala

```diff
@@ -0,0 +1,19 @@
+package net.wiringbits.repositories.daos
+
+import anorm.SqlStringInterpolation
+import net.wiringbits.repositories.models.Table
+
+import java.sql.Connection
+
+object TablesDAO {
+
+  def all()(implicit conn: Connection): List[Table] = {
+    SQL"""
+        SELECT table_name
+          FROM information_schema.tables
+        WHERE table_schema='public'
+          AND table_type='BASE TABLE';
```

Comment:
It is ideal that you always use a deterministic ordering, I have a [draft](https://github.com/AlexITC/notes/blob/master/development-good-practices.md#sql) for other useful hints